auth/kubernetes.go

```diff
@@ -0,0 +1,13 @@
+package auth
+
+import (
+	"os"
+)
+
+func getKubernetesToken() (string, error) {
+	tokenFile, err := os.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
```

Comment:
Add a check that file exists maybe? Or and argument from where to get the token?